_posts/en/newsletters/2019-04-09-newsletter.md

```diff
@@ -0,0 +1,309 @@
+sending support and notable merges in popular Bitcoin infrastructure
+projects.
+
+## Action items
+
+- **Help test Bitcoin Core 0.18.0 RC3:** The third Release Candidate
+  (RC) for the next major version of Bitcoin Core has been [released][0.18.0].
+  This may be the final test version, so we encourage organizations and
+  expert users to test it promptly if they want to ensure any
+  possible regressions are caught before release.  Please use [this
+  issue][Bitcoin Core #15555] for reporting feedback.
+
+- **Help test LND 0.6-beta RC3:** the first, second, and third RCs for
+  the next major version of LND were [released][lnd releases] this week.
+  Testing by organizations and experienced LN users is encouraged to
+  catch any regressions or serious problems that could affect users of
+  the final release.  [Open a new issue][lnd issue] if you discover any
+  problems.
+
+## News
+
+- **Discussion about an assumed-valid mechanism for UTXO snapshots:**
+  When Bitcoin Core developers are preparing a new major release,
+  one developer selects the hash of a recent block on the best block
+  chain.  Other well-known contributors check their personal nodes and
+  ensure that hash is indeed part of the best block chain, and then add
+  that hash to the software as the "assumed valid" block.  When new
+  users start Bitcoin Core for the first time, the program then defaults
+  to skipping verification of signatures on all transactions included
+  in blocks before the assumed valid block.  The program still keeps
+  track of the bitcoin ownership changes produced by each transaction in
+  a index called the Unspent Transaction Output (UTXO) set.  Although
+  reviewing each historic ownership change still takes time, simply
+  skipping signature checking reduces initial sync time by about 80%
+  according to [tests][0.14 tests].  Gregory Maxwell, who [implemented
+  the assumed valid feature][Bitcoin Core #9484], has argued that,
+  "Because the validity of a chain history is a simple objective fact,
+  it is [easy] to review this setting."
+
+    This week James O'Beirne started a [thread][assumeutxo thread] on
+    the Bitcoin-Dev mailing list about taking a hash of the UTXO set at
+    a particular block, having multiple well-known contributors verify
+    they get the same hash, and then having freshly installed Bitcoin
+    Core nodes default to using that hash to download the exact same
+    UTXO set.  This would allow a newly-started node to skip not only
+    signatures but all block chain data before the assumed valid block,
+    perhaps reducing the bandwidth and time requirements to start a node
+    today by 95% or more (and certainly more as the block chain
+    continues to grow).  This is an old idea, is currently implemented
+    in some software that uses Bitcoin Core, and is part of the
+    motivation for research into other techniques such as [fast
+    updatable UTXO commitments][] and [automatic levelDB backups][].
```

Comment:
James's proposal is to start from the assumed-UTXO block and validate to the tip, but then to 'back validate' from genesis to the assumed-UTXO block. Once that's done, there's no difference in the security model between that and a full node that has validated from genesis. I think it's important to include this in your summary.

Reply:
Thanks for catching this.  Somehow I skimmed right past that in my read of his proposal, even though I know it was part of earlier proposals such as Schnelli's.